src/dtos/user-dto.ts

```diff
@@ -0,0 +1,19 @@
+import { User } from '../entity/user';
+
+export class UserDto {
+    id: string;
+    oidcId: string;
+    provider: string;
```

Comment:
Presumably this uniquely identifies the IDP?
If so, please add a comment to this effect.
If not, add a different comment to different effect. ;-)